wordpress-mobile/WordPress-iOS
# Updated Domains success screen

This PR implements the success screen for the domains purchase flow. When rotating to landscape on a phone I opted to remove the illustration. When text gets very big, the illustration may be hidden (for 'accessibility' type sizes) and the view should scroll.

cc @kyleaparker

| Standard | XXL | Accessibility XXXL |
|---|---|---|
| ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 16 15 00](https://user-images.githubusercontent.com/4780/137350824-d0b68cfc-bc4a-40fa-ba5c-9bbf968b0a82.png) | ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 15 02 20](https://user-images.githubusercontent.com/4780/137351078-658d03ae-806a-46d8-9516-bbd4a679fa3a.png) | ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 15 02 33](https://user-images.githubusercontent.com/4780/137350993-70b411bc-b027-4101-a577-196d7041f272.png) |
| ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 16 15 03](https://user-images.githubusercontent.com/4780/137350802-8c87a815-a8f3-4946-99ef-7871ba7cc3f7.png) | ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 15 02 23](https://user-images.githubusercontent.com/4780/137351067-7d07a6cb-0add-4965-bd84-6c8aed776be5.png) | ![Simulator Screen Shot - iPhone 12 mini - 2021-10-14 at 15 02 38](https://user-images.githubusercontent.com/4780/137350962-2d9e327b-fbc1-4c7b-9fad-ca56dfa77c4c.png) |

## To Test

- Build and run
- Try both of the following:
    - Redeem a domain credit to use the native flow, and check you see this screen at the end
    - Add another domain to use the web flow and check you see this screen. I was unable to test this as I'm having issues again with the webview auto dismissing. Looking at the code, I _think_ it should be displayed though.
- Check with different dynamic font sizes

## Regression Notes

1. Potential unintended areas of impact

- This affects the existing domain flows, but this screen is an improvement over what we had already. Changes are limited to this new screen.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
